src/resources/armies/moloch.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
Hmm, we already added moloch before. Why is this commit adding it from scratch again?